### PR TITLE
add styles

### PR DESCRIPTION
**Pull request by berecik**
there is a style directory

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -46,6 +48,6 @@
     </form>
     <p id="result"></p>
 </main>
-<script src="events.js"></script>
```
Comment (berecik) [inline events.html:49]: Is the most stupid thing I ever see!!!

**Review by berecik — COMMENTED** [review]
fail fail fail!!!

File: events.html [hunk 2]
```diff
@@ -3,6 +3,8 @@
 <head>
     <meta charset="UTF-8">
     <title>Events</title>
+    <link href="css/style.css" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.js"></script>
 </head>
```
Comment (callicarpa) [inline events.html:8]: Ha ha ha, bardzo smieszne!

**Review by masliw — COMMENTED** [review]
Seriously???

File: events.html (hunk 1 above)
Comment (masliw) [reply events.html:49]: 4 me the same!

**Pull request merged**